packages/expo-maps/ios/ExpoMaps/GoogleMaps/GoogleMapsClusters.swift

```diff
@@ -1,4 +1,3 @@
-#if HAS_GOOGLE_UTILS
```

Comment:
@lukmccall - not sure if this macro is defined anywhere? I searched the codebase but I couldn't find it - it seems to have been arbitrarily listed here for the purposes of disabling the advanced google maps functionality.